shared/structures/src/members/PlatformMember.ts

```diff
@@ -96,6 +96,22 @@ export class PlatformFamily {
         return family;
     }
 
+    add(platformMember: PlatformMember) {
+        if (!this.belongsToFamily(platformMember.member)) {
+            console.error('PlatformMember does not belong to this family');
+            return;
+        }
```

Comment:
Ik zou dit weglaten nu is er gevaar op onverwachte bugs en moet je de interne logica kennen
- het zou op zijn minst een error moeten throwen, anders krijg je undefined behaviour 
- of de naam moet veranderd worden naar addIfBelongsToFamily (zou ik hier niet doen)

Reply:
Als ik de rest van je code bekijk, is dit ook niet nodig omdat je dit al checkt buiten deze methode waar je het gebruikt bij het importeren